int32 check_type
string coffee_type
---
string quantity

